REQUIRE

```diff
@@ -0,0 +1,2 @@
+julia 0.3
+Compat
```

Comment:
I tried to get the minimum bounds of least warnings,
because this PR has to against the deprecation warning.
😅